src/Servers/Kestrel/test/BindTests/AddressRegistrationTests.cs

```diff
@@ -578,7 +584,7 @@ public async Task ThrowsWhenBindingToIPv4AddressInUse()
         [IPv6SupportedCondition]
         public async Task ThrowsWhenBindingToIPv6AddressInUse()
         {
-            IgnoredCriticalLogExceptions.Add(typeof(IOException));
+            ThrowOnCriticalErrors = false;
```

Comment:
Why change this approach?

Reply:
There is no exception logged for this even anymore that is why the `IgnoredCriticalLogExceptions` will not work. I'm checking the message content instead.